src/AcceptanceTests/NServiceBus.AzureServiceBus.AcceptanceTests.csproj

```diff
@@ -11,7 +11,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="WindowsAzure.ServiceBus" Version="5.*" />
+    <PackageReference Include="WindowsAzure.ServiceBus" Version="5.0" />
```

Comment:
Once 5.0.2 is out, assuming it's fixes the issue, we should revert this change.

Reply:
In general, we try to avoid now wild cards since they are too dangerous (breaking the builds suddenly)